pallets/data-preservers/src/lib.rs

```diff
@@ -168,143 +318,53 @@ pub mod pallet {
 
         UnknownProfileId,
         NextProfileIdShouldBeAvailable,
+
+        /// Made for `AssignmentPayment` implementors to report a mismatch between
+        /// `ProviderRequest` and `AssignerParameter`.
+        AssignmentPaymentRequestParameterMismatch,
+
+        ProfileAlreadyAssigned,
+        ProfileNotAssigned,
+        ProfileIsNotElligibleForParaId,
+        WrongParaId,
+        MaxAssignmentsPerParaIdReached,
     }
 
     #[pallet::composite_enum]
     pub enum HoldReason {
         ProfileDeposit,
     }
 
+    #[deprecated]
     #[pallet::storage]
-    #[pallet::getter(fn boot_nodes)]
     pub type BootNodes<T: Config> = StorageMap<
         _,
         Blake2_128Concat,
         ParaId,
-        BoundedVec<BoundedVec<u8, T::MaxBootNodeUrlLen>, T::MaxBootNodes>,
+        BoundedVec<BoundedVec<u8, T::MaxNodeUrlLen>, T::MaxAssignmentsPerParaId>,
         ValueQuery,
     >;
 
     #[pallet::storage]
+    #[pallet::getter(fn profiles)]
```

Comment:
```suggestion
```

Parity is deprecating getters, so we shouldn't add new ones

Reply:
let's make sure whenever we do this that the storages are public please